src/vm/excep.cpp

```diff
@@ -7314,8 +7318,8 @@ AdjustContextForWriteBarrier(
 {
     WRAPPER_NO_CONTRACT;
 
+#ifndef WIN64EXCEPTIONS
 #ifdef _TARGET_X86_
```

Comment:
@jkotas Could you let me know some details? I thought that ``WIN64EXCEPTIONS`` is a exception handling ABI based on C++ try/catch and libunwind, but it seems wrong.